doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,274 @@
+.. ug_creating_matter_accessory: 
+
+Creating Matter accessory
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+| This tutorial shows how to create a custom device application using Matter.     
+| It will show a way to modify Matter Template sample in order to obtain a sensor application.
+
+The goal of this guide is to create a sensor, which, if active, periodically updates measured value.
+
+
+Matter Template sample
+**********************
+
+| The base for buiding a sensor device will be a Matter Template sample which provides a basic implementation of a Matter accessory.
+| First step in creating sensor device will be to copy content from :file:`samples/matter/template` to another place eg. :file:`samples/matter/sensor`.
+
+From now on, paths will be given relative to :file:`samples/matter/sensor` directory.
+
+Application Clusters
+********************
+
+Adding new functionalities to a Matter sample requires adding new application clusters.         
+
+Cluster is a representation of a single functionality within a Matter accessory.
+Each cluster contains attributes which are stored in device's memory, and commands that can be used to modify or read the state of the device, including the cluster attributes.
```

Comment:
I'm not sure about this changing device state. It's somehow confusing to me. How about writing just:
```suggestion
Each cluster contains attributes which are stored in device's memory, and commands that can be used to modify or read those attributes values.
```

Reply:
But that's not exactly right. Commands may or may not be related to the attributes. Maybe Greg will come up with a less confusing way to describe that fact.

Reply:
Ok, if that is what you mentioned, I'm fine with that. I assumed that we want to mainly focus on reading attributes values, so calling it reading the device state was for me going too far. I think it can stay as it is.